conman/redirects/models.py

```diff
@@ -13,9 +14,11 @@ class RouteRedirect(Route):
     This model holds the data required to make that connection.
     """
     handler = handlers.RouteRedirectHandler.path()
-    target = models.ForeignKey('routes.Route', related_name='+')
+    target = models.ForeignKey('routes.Route', related_name='+', on_delete=models.CASCADE)
     permanent = models.BooleanField(default=False, blank=True)
 
+    objects = RouteManager()
```

Comment:
This seemed to me the cleanest option to handle [the inheritance changes planned for Django 2.0](https://docs.djangoproject.com/en/1.10/releases/1.10/#manager-use-for-related-fields-and-inheritance-changes).

I tried the `Meta.manager_inheritance_from_future=True` option, but that didn't play well with `makemigrations` on earlier Django versions.

We may want to explicitly document something around this.